.gitlab-ci.yml

```diff
@@ -32,13 +32,13 @@ stages:
         --create-namespace
         --namespace $KUBE_NAMESPACE
         --set image.backend.repository="${BACKEND_CONTAINER_REPO}"
-        --set image.backend.tag="${CI_COMMIT_SHORT_SHA}-beta"
+        --set image.backend.tag="${CI_COMMIT_SHORT_SHA}-beta-leaktests"
         --set image.frontend.repository="${FRONTEND_CONTAINER_REPO}"
-        --set image.frontend.tag="${CI_COMMIT_SHORT_SHA}-beta"
+        --set image.frontend.tag="${CI_COMMIT_SHORT_SHA}-beta-leaktests"
         --values values-parity.yaml
         $KUBE_NAMESPACE parity/substrate-telemetry
   rules:
-    - if: '$CI_COMMIT_BRANCH == "master"'
+    - if: '$CI_COMMIT_BRANCH == "jsdw-mem-testing"'
```

Comment:
This is the only change that should happen.
I would suggest to preserve the `'$CI_COMMIT_BRANCH == "master"'` and only add ` '$CI_COMMIT_BRANCH == "jsdw-mem-testing"'` to the rules.

Reply:
Ah ok; as it stands, it doesn't seem to be running in Gitlab anyway, but happy to add the "master" back; I wasn't intending on merging this PR anywhere though (unless it's a requirement that I merge something to master to have it work at all?)

Reply:
tbh I'm not sure about your question, But I think it should be merged to the master to work properly. Maybe it's better to ask @TriplEight or @rcny .